lib/mix/tasks/docs.ex

```diff
@@ -88,6 +88,15 @@ defmodule Mix.Tasks.Docs do
 
     * `:groups_for_extras`, `:groups_for_modules`, `:groups_for_functions` - See next sections
 
+    * `:group_modules_by_nesting` - A list of atoms (e.g. `Foo.Bar`) that will be truncated
```

Comment:
Minor nitpick: should we say here "A list of module names (e.g. `Foo.Bar`)"? The example says `Foo.Bar` instead of `:"Elixir.Foo.Bar"`

Reply:
Fixed.